README.md

```diff
@@ -1 +1,185 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project"></a>
+
+**[Hello world]**  is about html and css and javascript. It
+is also apply linters to support code optimization
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+This project is based on HTML, CSS, JAVASCRIPT, Linters
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Glossary/HTML5">Html5</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://github.com/microverseinc/linters-config">Linters</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">None</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="">None</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Responsive]**
+- **[Dynamic]**
+- **[Professional]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+none
+
+### Setup
+Clone this repository to your desired folder:
+
+### Install
+
+none
+
+### Usage
+
+Clone it
+
+### Run tests 
+none
+
+### Deployment
+none
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Than Trong Khanh**
+
+- GitHub: [khanh-devos](https://github.com/khanh-devos)
+- Twitter: [@khanhror](https://twitter.com/home?lang=en)
+- LinkedIn: [Khanh RoR](https://www.linkedin.com/in/khanh-than-trong-3b4344246/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[add some specific content to it]**
+- [ ] **[implement responsive]**
+- [ ] **[use javascript to add some dynamic feature]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, can invite me a cup of tea
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+If you like this project, can invite me a cup of tea
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Ouch this is so close but it seems like the MIT link is broken check the video below:
  ![firefox_nHQ4CD7Fe6](https://user-images.githubusercontent.com/104968236/234548735-ee38443d-0903-4526-bfa6-b6f9fabd387e.gif)
- So this issue is occurring because you don't have a LICENSE file at your root repository, you can add the MIT.md file [found here,](https://github.com/microverseinc/readme-template/blob/master/MIT.md), customize the content and replace the `./LICENSE` in line `183` with `./MIT.md` 
```suggestion
This project is [MIT](./MIT.md) licensed.
```

Reply:
Yeah !!!, thank you a lot, Mr.Abel